contexts/resource/index.json

```diff
@@ -20,18 +40,18 @@
     "_rev": "nxv:rev",
     "_distribution": {
       "@container": "@set",
```

Comment:
because if it is not a set en you have only one element it will expand to
```json
{
   "_distribution": {
      ...
   }
}
```
note the {} instead of the expected []

Reply:
The @set container will only ensure the uniqueness and order of the distributions items when expanded:

The distribution items will always be kept I think:

[With @set](https://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%7B%22%40vocab%22%3A%22https%3A%2F%2Fschema.org%22%2C%22dcat%22%3A%22http%3A%2F%2Fwww.w3.org%2Fns%2Fdcat%23%22%2C%22_distribution%22%3A%7B%22%40id%22%3A%22dcat%3Adistribution%22%2C%22%40container%22%3A%22%40set%22%7D%7D%2C%22_distribution%22%3A%5B%7B%22description%22%3A%22distribution%201%22%7D%2C%7B%22description%22%3A%22distribution%202%22%7D%5D%7D):



[Without @set](https://json-ld.org/playground/#startTab=tab-expanded&json-ld=%7B%22%40context%22%3A%7B%22%40vocab%22%3A%22https%3A%2F%2Fschema.org%22%2C%22dcat%22%3A%22http%3A%2F%2Fwww.w3.org%2Fns%2Fdcat%23%22%2C%22_distribution%22%3A%7B%22%40id%22%3A%22dcat%3Adistribution%22%7D%7D%2C%22_distribution%22%3A%5B%7B%22description%22%3A%22distribution%201%22%7D%2C%7B%22description%22%3A%22distribution%202%22%7D%5D%7D)

Reply:
Yes. What I meant is (from the [W3C Json-LD](https://json-ld.org/spec/latest/json-ld/#sets-and-lists)
> The use of @set in the body of a JSON-LD document is optimized away when processing the document, as it is just syntactic sugar. However, @set is helpful when used within the context of a document. Values of terms associated with an @set or @list container are **always represented in the form of an array, even if there is just a single value that would otherwise be optimized to a non-array form in compact form**

Reply:
great.